### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-ru/content_questlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#297]
```
translated quests v069, v0610, v0611.
vilegard  and feygard renamed to transliterated names.
```

Original issue reported on code.google.com by `e.solodo...@gmail.com` on 25 Jan 2012 at 8:40

Attachments:
- [content_questlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-297/comment-0/content_questlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Issue 296 has been merged into this issue.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r223.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 9:30
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
